content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1339,7 +1339,7 @@ In the following example, `authorId` is a both a relation scalar and the ID of `
 
 ```prisma
 model Profile {
-  authorId Int    @id
+  authorId Int    @id @unique
```

Comment:
Want to check this one - do we still add the `@unique` attribute when the relation scalar already has an `id` attribute

Reply:
So fine without:
```suggestion
  authorId Int    @id
```